static/styles/landing-page.css

```diff
@@ -343,24 +343,14 @@ nav ul li.active::after {
 
     padding-top: 120px;
 
-    transition: all 0.5s ease;
-    -webkit-transform: translateY(120px);
-    transform: translateY(120px);
+    transition: opacity 0.5s ease;
```

Comment:
At least for me, clicking around the menu between the different items, it feels slow, and usually that's a bad thing.  I think you're right that it's good to have animations here, but maybe 0.5s is on the long side?